### PR TITLE
Update WopiValidator Documentation for iOS

### PR DESCRIPTION
**Pull request by mandaryadav**
Update WopiValidator Documentation for iOS

### CODE REVIEW

File: docs/native/src/testing/validator.rst [hunk 1]
```diff
@@ -1,3 +1,15 @@
 
-|stub-icon| Using the validator app to validate your WOPI implementation
-========================================================================
+Using the validator application to validate your WOPI implementation
+====================================================================
+
+Since WOPI is used in both the Office Online and Office for iOS integrations, you can verify your WOPI implementation by following
```
Comment (tylerbutler) [inline docs/native/src/testing/validator.rst]: Use the `|wac|` and `|Office iOS|` tokens instead of hard coding the names.

File: docs/native/src/testing/validator.rst [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+the instructions at :ref:`validator`
```
Comment (tylerbutler) [inline docs/native/src/testing/validator.rst]: nit: missing period

File: docs/native/src/testing/validator.rst [hunk 3]
```diff
@@ -1,3 +1,15 @@
...
+
+If you do not integrate with Office Online, you can still verify your WOPI implementation using the above instructions by building
+a minimal :term:`host page` and setting the :term:`VALIDATOR_TEST_CATEGORY` to ``OfficeNativeClient``. This will run the
+WopiValidator application in a mode that will run only the tests which are necessary for Office for iOS integration.
```
Comment (tylerbutler) [inline docs/native/src/testing/validator.rst]: I'd rephrase this to:

> This will run only the tests that are necessary for `|Office iOS|` integration.

**Comment by tylerbutler** [comment]
Looks good overall, just a few tweaks needed.

**Comment by mandaryadav** [comment]
@tylerbutler : Addressed the feedback. Let me know if I missed something.

**Comment by mandaryadav** [comment]
Thanks Tyler !
